shell/assets/translations/zh-hans.yaml

```diff
@@ -6781,10 +6965,10 @@ legacy:
   globalDnsEntries: 全局 DNS 条目
   globalDnsProviders: 全局 DNS 提供商
   istio: Istio
-  logging: 日志管理
+  logging: Logging
   notifiers: Notifiers
   monitoring: 监控
-  pipelines: 流水线
+  pipelines: 管道
```

Comment:
"流水线" should be a reasonable expression.
I can get the desired results if I google "Rancher 流水线"

Reply:
I think both `管道` and `流水线` should be fine in terms of the meaning. I changed it to `管道` because I found that Azure uses `管道` for `pipeline` and AWS uses both `管道` and `流水线` (maybe `管道` is used more often). If we decide to use `流水线`, I can change it back and change all `流水线` to `管道` in rancher 2.6 documentation as well to ensure we use the same translation for the same term.

Reply:
Let's keep the `流水线`, users are used to the concept, it's been around for a long time in Rancher.